backend/HitMeApp/HitMeApp/Program.cs

```diff
@@ -1,19 +1,41 @@
 using Autofac.Extensions.DependencyInjection;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Hosting;
+using Serilog;
+using System;
 
 namespace HitMeApp.Api
 {
     public class Program
     {
         public static void Main(string[] args)
         {
-            CreateHostBuilder(args).Build().Run();
+            var messageTemplate = "[{Timestamp:HH:mm:ss} {Level:u3}] [{Module}] {Message:lj}{NewLine}{Exception}";
```

Comment:
If this will stay like this for longer period we could introduce const here for message template. But I have suspicion that this is just temporary.

Reply:
Nope. It will stay this way. I just didn't want for it to overflow the editor's viewport.
IMHO it doesn't really matter if it's const or not as it only serves the purpose of being passed to the builder.